src/components/styles/demo/demo.module.css

```diff
@@ -0,0 +1,47 @@
+.demo-section-top {
+  background-color: var(--cornslik);
+  height: 50%;
```

Comment:
height jest nam potrzebne na pewno?

Reply:
wydaje mi się, że tak, jak zobaczysz stronkę to zobaczysz dlaczego